arangod/RestHandler/RestMetricsHandler.cpp

```diff
@@ -152,7 +149,17 @@ RestStatus RestMetricsHandler::execute() {
     return RestStatus::DONE;
   } else {
     std::string result;
-    metrics.toPrometheus(result);
+    auto mode = metrics::CollectMode::Local;
+    if (it = values.find("mode"); it != values.end()) {
```

Comment:
I'd also prefer to fail the request in case if invalid mode is specified

Reply:
Ok